### PR TITLE
Various improvements

### PR DESCRIPTION
**Pull request by HypeMC**

### CODE REVIEW

File: Tests/Fixtures/FixtureFactory.php [hunk 1]
```diff
@@ -56,6 +56,7 @@ final class FixtureFactory
     public const FIXTURE_SCOPE_SECOND = 'rock';
 
     public const FIXTURE_USER = 'user';
+    public const FIXTURE_PASSWORD = 'password';
```
Comment (X-Coder264) [inline Tests/Fixtures/FixtureFactory.php]: @HypeMC I don't see this being used anywhere?

Comment (HypeMC) [reply Tests/Fixtures/FixtureFactory.php]: @X-Coder264 It was add with https://github.com/trikoder/oauth2-bundle/pull/18/files#diff-a87d1acb0e93f0f5a6dd7abd960527e3 , the tests pass without it, but I figured having it doesn't hurt.

File: Tests/Unit/InMemoryRefreshTokenManagerTest.php [hunk 2]
```diff
@@ -20,15 +20,17 @@ public function testClearExpired(): void
 
         timecop_freeze(new DateTime());
 
-        $testData = $this->buildClearExpiredTestData();
+        try {
+            $testData = $this->buildClearExpiredTestData();
```
Comment (X-Coder264) [inline Tests/Unit/InMemoryRefreshTokenManagerTest.php]: @HypeMC Does this line (and similar lines in other tests) need to be inside the `try` block? I don't see a case where building the data can throw an exception.

Comment (HypeMC) [reply Tests/Unit/InMemoryRefreshTokenManagerTest.php]: @X-Coder264 Probably not, in theory, `new DateTime` could throw an exception if an invalid value is passed, but this is more for consistency I guess, everything between free & return are in a try..finally block.

**Pull request merged**